package.json

```diff
@@ -65,6 +65,7 @@
     "es6-error": "4.1.1",
     "event-to-promise": "0.8.0",
     "firefox-profile": "1.3.1",
+    "fs-extra": "9.0.1",
```

Comment:
I'm not keen on adding new dependencies because it generally results in more maintenance load (e.g. having to publish new versions when a security issue is found in the dependency). But since we're already indirectly depending on this module, I'm fine with it.

In a follow-up issue/PR we can probably replace `mz` with this, since `mz`'s functionality overlaps with `fs-extra`.